packages/ses/docs/lockdown.md

```diff
@@ -366,12 +367,54 @@ the container to exit explicitly, and we highly recommend setting
 - `'none'`: do not install traps for uncaught exceptions. Errors are likely to
   appear as `{}` when they are reported by the default trap.
 
+## `evalTaming` Options
+
+This option only affects the initial compartment!
+
+**Background**: For every compartment including the implied compartment in the
+initial JavaScript realm, there are evaluators `eval` and `Function`.
+The default lockdown behavior isolates all of these evaluators.
+
+Replacing the realm's initial evaluators is not necessary to ensure the
+isolation of guest code because guest code must not run in the initial realm.
+However, replacing these evaluators is the safest and therefore default
+evaluator taming option (`"safeEval"`) because it may limit the harm that guest code can cause if it escapes its assigned compartment.
+
+However, only the exact `eval` function from the initial realm can be used to
+perform direct-eval, which runs in the lexical scope in which the direct-eval syntax appears (direct-eval is a special form rather than a function call).
+The SES shim itself uses direct-eval internally to construct an isolated
+evaluator, so replacing the initial `eval` prevents any subsequent program
+from using the same mechanism to isolate a guest program.
+
+The `"unsafeEval"` option for `evalTaming` leaves the original `eval` in place
+for other isolation mechanisms like isolation code generators that work in
+tandem with SES.
+This option may be useful for web pages with a Content Security Policy
+that excludes `unsafe-eval` or browser extension environments with similar
```

Comment:
This seems contradicted by
> ```js
> lockdown({ evalTaming: 'unsafeEval' });
> // !!!! DO NOT USE IT IN PRODUCTION WITH "unsafe-eval" in CSP or even no CSP !!!!
> ```

below

Reply:
> web pages with a Content Security Policy that excludes `unsafe-eval` or browser extension environments

I think it should be "includes" other than "excludes"

Reply:
or "allows `unsafe-eval`"